src/components/Global/LogStatusButton/index.tsx

```diff
@@ -1,13 +1,23 @@
 import { Button } from '../index';
 import { signIn, signOut } from 'next-auth/react';
 import React from 'react';
+import { LogStatusButtonProps } from '../../../types/logStatusButton';
 
-const LogStatusButton = ({ status }) => {
-  return status === 'authenticated' ? (
-    <Button style={'flat'} content={'Logout'} onClick={() => signOut()} />
-  ) : (
-    <Button style={'flat'} content={'Login'} onClick={() => signIn()} />
-  );
+const LogStatusButton = ({ status, usedInPage }: LogStatusButtonProps) => {
+  if (usedInPage === 'home') {
```

Comment:
The logic should be cleaned up in general, but also why is this a string parameter? Aren't there only 2 options

Logic should be cleaned up so the "logout" button is only written once in the code

Reply:
I think we can pass a parameter named link and link whatever link we want to and also can be used to check if need signin() on click

Reply:
This feels more complicated, I think let's just stick with the proposal I made to get this merged